release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -236,6 +236,24 @@ For more information, see xref:../nodes/pods/nodes-pods-priority.html#non-preemp
 === Developer experience
 
 
+
+[id="ocp-4-7-insights-operator"]
+=== Insights Operator
+
+In {product-title} 4.7, the Insights Operator collects the following additional information:
+
+* The top 100 `InstallPlan` entries to identify invalid Operator Lifecycle Manager (OLM) installations
+* The service accounts from the Kubernetes default namespace and the `openshift*` built-in namespaces
+* The `ContainerRuntimeConfig` and `MachineConfigPools` configuration files to verify container storage limits
+* The configuration files for all available `operator.openshift.io` control pane resources to identify operators in unmanaged states
+* The `NetNamespaces` names, including their `netID` and egress IP addresses
+* The `StatefulSet` configuration definitions from the OpenShift default namespaces to check the Prometheus monitoring storage type
```

Comment:
Does it make sense to change OpenShift &rarr; {product-title}?

Reply:
Updated.